src/main/java/net/sf/jabref/migrations/PreferencesMigrations.java

```diff
@@ -79,4 +88,43 @@ public static void upgradeSortOrder() {
         }
     }
 
+    /**
+     * Migrate LabelPattern configuration from version 3.3-3.5 to new BibtexKeyPatterns
```

Comment:
Should be possible. I'll implement the logic that the newest available prefs are migrated to 3.6